docs/sdk/authentication/session-sigs/intro.md

```diff
@@ -11,116 +11,63 @@ import FeedbackComponent from "@site/src/pages/feedback.md";
 `SessionSigs` are only available on Ethereum and are heavily in development, so things may change. Be sure to use the latest version of the Lit JS SDK.
 
 :::
+Session signatures are how the Lit network authenticates your connection, verifies your identity, and confirms your capabilities. 
```

Comment:
i think we could simplify this a bit. Perhaps someting along the lines of "Session signatures are used to authenticate with the Lit nodes and create a secure connection to the Lit network"

Reply:
I've been capitalizing Lit jargon since it's a specific thing in our ecosystem. My preference is to always capitalize Session Signatures